pkg/controller/registry/resolver/cache.go

```diff
@@ -9,12 +9,12 @@ import (
 	"time"
 
 	"github.com/blang/semver"
+	v1alpha1listers "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/listers/operators/v1alpha1"
```

Comment:
import format nit

Reply:
If you search the OLM code for `github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/listers/operators/v1alpha1`, it's typically imported as `v1alpha1, listers, or operatorslisters`, which `v1alpha1` being the most common approach.

Reply:
in here i think its ok to convert to v1alpha1, but there is another place that as v1alpha1 already for another package.

Reply:
`olmv1alpha1` should be solid enough to avoid confusion. 
At the end of the day we should adopt a convention that is followed throughout the entire project, but that's a discussion for later.

Reply:
We don't even need to give it a special name - no other import in this file users `v1alpha1`.